src/App.jsx

```diff
@@ -12,59 +13,79 @@ export const goods = [
   'Honey',
   'Jam',
   'Garlic',
-];
+].map((item, index) => ({
+  id: index + 1,
+  name: item,
+}));
```

Comment:
DON'T use index as a key. Using indexes as keys is not recommended because it can negatively affect performance and may cause issues with component state. You should use a unique identifier that does not change. If the items in your list have unique properties, use those instead.